src/exportDataAction/ExportDataAction.js

```diff
@@ -66,8 +69,27 @@ class ExportDataAction {
     }
   }
 
+  historicalFilterString() {
+    const sessionFilter = this.sessionService.getHistoricalSessionFilter();
+
+    if (!sessionFilter) {
+      return '';
+    }
+
+    const forFilename = true;
+
+    return `${sessionFilter.host}_${formatNumberSequence(sessionFilter.numbers, forFilename)}`;
```

Comment:
Prefer not to extend functions using boolean flags. Can you just transform the string after calling `formatNumberSequence`?

Reply:
This is the route I went.